app/assets/javascripts/utilities/timeAgo.js

```diff
@@ -25,15 +25,34 @@ function secondsToHumanUnitAgo(seconds) {
   return wholeUnits + ' ' + unitName + ' ago';
 }
 
-function timeAgo(oldTimeInSeconds, maxDisplayedAge = 60 * 60 * 24 - 1) {
+/**
+ * Returns a given time in seconds as a human readable form, e.g. (5 min ago)
+ *
+ * @param {object} options
+ * @param {number} options.oldTimeInSeconds
+ * @param {function} [(humanTime) =>
+      `<span class="time-ago-indicator">(${humanTime})</span>`] options.formatter
+ * @param {number} [60 * 60 * 24 - 1] options.maxDisplayedAge The maximum display age in seconds
+ *
+ * @returns {string} A formatted string in human readable form. Note that the default formatter returns a string with markup in it.
+ */
+function timeAgo({
+  oldTimeInSeconds,
+  formatter = humanTime =>
+    `<span class="time-ago-indicator">(${humanTime})</span>`,
+  maxDisplayedAge = 60 * 60 * 24 - 1,
+}) {
   const timeNow = new Date() / 1000;
   const diff = Math.round(timeNow - oldTimeInSeconds);
 
   if (diff > maxDisplayedAge) return '';
 
-  return (
-    "<span class='time-ago-indicator'>(" +
-    secondsToHumanUnitAgo(diff) +
-    ')</span>'
-  );
+  return formatter(secondsToHumanUnitAgo(diff));
+}
+
+// TODO: This is for Storybook/jest.
+// Longterm, this should be a utility function that can be imported.
+// For the time being, duplication of this function is being avoided.
+if (typeof globalThis !== 'undefined') {
```

Comment:
The version of node we use, currently 12.x, supports `globalThis`, so this is all good for jest. In regards to Storybook, I made the call, that anyone using or viewing Storybook will be on a modern enough browser. All Chromium-based browsers support `globalThis` as does Firefox.